parlai/crowdsourcing/tasks/dialcrowd/run.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+from dataclasses import dataclass, field
+from typing import Any, List
+
+import hydra
+from mephisto.operations.hydra_config import register_script_config
+from omegaconf import DictConfig
+
+from parlai.crowdsourcing.tasks.dialcrowd.dialcrowd_blueprint import (
+    STATIC_BLUEPRINT_TYPE,
+)
+from parlai.crowdsourcing.tasks.turn_annotations_static.util import run_static_task
```

Comment:
> @jessicah25 I'm assuming you meant to import from your version of this util, as included in this PR?
> 
> @EricMichaelSmith as an aside, I imagine if this util is usable outside of just `turn_annotations_static` it could be good to pull out in a general form, but I'm not positive that this is perfectly generalizable even here.

Good point, making a note to look into this

Reply:
> @jessicah25 I'm assuming you meant to import from your version of this util, as included in this PR?
> 
> @EricMichaelSmith as an aside, I imagine if this util is usable outside of just `turn_annotations_static` it could be good to pull out in a general form, but I'm not positive that this is perfectly generalizable even here.

Good point, making a note to look into this